cmd/cloud/key.go

```diff
@@ -83,7 +83,11 @@ func newListKeyCommand() *cobra.Command {
 		Use:   "list",
 		Short: "List API keys",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := executeListKeys()
+			err := getNewAccessTokenUsingRefreshTokenIfCurrentIsExpired()
```

Comment:
Looks like `getNewAccessTokenUsingRefreshTokenIfCurrentIsExpired` is actually mutating an internal state ie. updating access token. The name is mislead because `getXXX` usually implies read operation.

May be something like `refreshAccessToken` is more intention revealing. The logic of caching / checking if current one is expired can be internal to this function.